app/flatpak-builtins-history.c

```diff
@@ -44,12 +44,14 @@ static char *opt_since;
 static char *opt_until;
 static gboolean opt_reverse;
 static const char **opt_cols;
+static gboolean opt_show_appstream_pulls;
 
 static GOptionEntry options[] = {
   { "since", 0, 0, G_OPTION_ARG_STRING, &opt_since, N_("Only show changes after TIME"), N_("TIME") },
   { "until", 0, 0, G_OPTION_ARG_STRING, &opt_until, N_("Only show changes before TIME"), N_("TIME") },
   { "reverse", 0, 0, G_OPTION_ARG_NONE, &opt_reverse, N_("Show newest entries first"), NULL },
   { "columns", 0, 0, G_OPTION_ARG_STRING_ARRAY, &opt_cols, N_("What information to show"), N_("FIELD,…") },
+  { "show-appstream-pulls", 0, 0, G_OPTION_ARG_NONE, &opt_show_appstream_pulls, N_("Show appstream pulls"), NULL },
```

Comment:
Oh also this is a clumsy name & description.

Reply:
`--all` might be another possibility, but the meaning here would differ from the meaning in `flatpak list` and `flatpak remote-ls`

Reply:
I think `--show-appstream-pulls` makes sense, and I think the man page could point the user to consider including the ref column, *and* we could also include it by default when `--show-appstream-pulls` is used?